convex_plane_decomposition_msgs/PlanarRegion[] contact_set
int32 duration